<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarginsAndBorders</class>
 <widget class="QDialog" name="MarginsAndBorders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>97</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>97</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Margins and Borders</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dpResources.qrc">
    <normaloff>:/program/images/icon</normaloff>:/program/images/icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelMargin">
       <property name="text">
        <string>Margin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMargin">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMargin"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelBorder">
       <property name="text">
        <string>Border:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxBorder"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxBorder"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="dpResources.qrc"/>
 </resources>
 <connections/>
</ui>
